BMM.Core/NewMediaPlayer/PlayerErrorHandler.cs

```diff
@@ -19,6 +20,8 @@ public class PlayerErrorHandler : IPlayerErrorHandler
         private const string ErrorPlayerLiveRadioStopped = Translations.MediaPlayer_ErrorPlayerLiveRadioStopped;
         private const string ErrorPlayerLiveRadioTooEarly = Translations.MediaPlayer_ErrorPlayerLiveRadioTooEarly;
 
+        private const int MaximumParameterLength = 120;
```

Comment:
according to this we could increase that to 125
https://github.com/MicrosoftDocs/appcenter-docs/issues/969